src/Recyclarr/Recyclarr.csproj

```diff
@@ -16,6 +16,13 @@
     <PackageReference Include="YamlDotNet" />
   </ItemGroup>
 
+  <!-- Following found during vulerabilities Code Scan  -->
+  <ItemGroup>
+    <PackageReference Include="System.Net.Http" />
+    <PackageReference Include="System.Private.Uri" />
+    <PackageReference Include="System.Net.RegularExpressions" />
```

Comment:
As mentioned in the other comment, this will need to change to `System.Text.RegularExpressions` as well.

Reply:
I fixed this in the latest push